### PR TITLE
Covalence

### PR DESCRIPTION
**Pull request by dco9**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
First version of Covalence integrations.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by adi88d** [comment]
@dco9 Please register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA) before we proceed with the review.

**Review by adi88d — CHANGES_REQUESTED** [review]
@dco9 Thank you for your contribution, few notes:
- Please add human readable to all the commands
- remove all the secrets from the code(look in the secrets section here: https://app.circleci.com/pipelines/github/demisto/content/94932/workflows/307a4ba6-d50b-4fe1-bf30-d8fa31dbcc14/jobs/386593)
- make sure all the unitises are passed(https://app.circleci.com/pipelines/github/demisto/content/94932/workflows/307a4ba6-d50b-4fe1-bf30-d8fa31dbcc14/jobs/386592)
- code must pass linters, execute[ lint command](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/lint/README.md)
- update the commands outputs - use[ json-to-outputs command](https://github.com/demisto/demisto-sdk/tree/master/demisto_sdk/commands/json_to_outputs), some outputs paths are wrong(for example `Covalence.Sensors..listeningInterfaces`)
- update outputs descriptions(for example `description: T` is not enough)
- Logo should match the requirements (120x50 px PNG, transparent background, <10kB in Size)

see my other comments below

File: Packs/CovalenceForSecurityProviders/Integrations/CovalenceForSecurityProviders/CovalenceForSecurityProviders.py [hunk 1]
```diff
@@ -0,0 +1,779 @@
...
+''' EXECUTION '''
+
+def main():
+    LOG(f'{demisto.command()} is called')
+    try:
+        if demisto.command() == 'test-module':
+            if get_health():
+                return_results('ok')
+            else:
+                return_results('nok')
```
[the remark this answers is not in the document]
Comment (dco9) [reply Packs/CovalenceForSecurityProviders/Integrations/CovalenceForSecurityProviders/CovalenceForSecurityProviders.py:627]: When the password is incorrect, an exception is raised:

image

File: Packs/CovalenceForSecurityProviders/Integrations/CovalenceForSecurityProviders/CovalenceForSecurityProviders.py [hunk 2]
```diff
@@ -0,0 +1,779 @@
...
+        return all(health_check_resp)
+    else:
+        return send_request('GET', '/rest/v1/health')
+
+def list_sensors():
+    target_org = demisto.args().get('target_org', None)
+
+    r = send_request('GET', '/rest/v1/sensors', target_org=target_org)
+
+    details = demisto.args().get('details', 'false')
```
[the remark this answers is not in the document]
Comment (dco9) [reply Packs/CovalenceForSecurityProviders/Integrations/CovalenceForSecurityProviders/CovalenceForSecurityProviders.py:291]: When running tests, I get:

`
NameError: name 'argToBoolean' is not defined
`

What am I doing wrong ?

[the remark this answers is not in the document]
Comment (adi88d) [reply Packs/CovalenceForSecurityProviders/Integrations/CovalenceForSecurityProviders/CovalenceForSecurityProviders.py:291]: add `from CommonServerPython import *` to the imports section

**Comment by dco9** [comment]
Closing to solve git username issue, see PR #13577

**Pull request closed without merging**